docs/src/language_servers.md

```diff
@@ -311,8 +311,29 @@ Follow installation instructions on [LSP-promql](https://github.com/prometheus-c
 
 ## PHP
 
+There are multiple options:
+
+### Intelephense
+
 Follow installation instructions on [LSP-intelephense](https://github.com/sublimelsp/LSP-intelephense).
 
+### Phpactor
+
+1. Install [Phpactor globally](https://phpactor.readthedocs.io/en/master/usage/standalone.html#installation-global).
+2. Open `Preferences > Package Settings > LSP > Settings` and add the `"phpactor"` client configuration to the `"clients"`:
+
+    ```json
+    {
+        "clients": {
+            "reason": {
```

Comment:
This says `”reason”`, typo?

Reply:
Oops, I forgot to rename that, see commit b0c1045862601ee6e6fd3250d3a441d46a6e8d7c